src/main/java/vinegar/ui/Ui.java

```diff
@@ -0,0 +1,78 @@
+package vinegar.ui;
+
+import vinegar.task.TaskList;
+import vinegar.task.Task;
+
+import java.util.Scanner;
+
+public class Ui {
+    private final Scanner scanner;
+
+    public Ui() {
+        scanner = new Scanner(System.in);
+    }
+
+    public String readCommand() {
+        return scanner.nextLine();
+    }
+
+    public void showWelcome() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm vinegar.Vinegar\n" + "What Can I do for you?");
+        System.out.println("____________________________________________________________");
```

Comment:
Should they be on the same indentation ?

Reply:
Can you clarify on this?